app/src/main/java/nl/tudelft/b_b_w/controller/ConversionController.java

```diff
@@ -8,21 +8,25 @@
  */
 public class ConversionController {
     // Variables which we need to create a hashed key
-    private String keyOwner;
+    private String blockOwner;
```

Comment:
Because when you use keyOwner you are refering to the PublicKeyOwner. This is totally not correct. This should be changed to blockOwner to prevent confusion.